extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -252,9 +252,38 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
         sortFunction
       );
 
+      /**
+       * naturalizes the dataset, and adds a retrieve bulkdata method
+       * to any values containing BulkDataURI.
+       * @param {*} instance
+       * @returns naturalized dataset, with retrieveBulkData methods
+       */
+      const addRetrieveBulkData = instance => {
+        const naturalized = naturalizeDataset(instance);
+        Object.keys(naturalized).forEach(key => {
+          const value = naturalized[key];
+          if (value && value.BulkDataURI && !value.Value) {
+            // Provide a method to fetch this
+            value.retrieveBulkData = () => {
+              const options = {
+                multipart: false,
+                BulkDataURI: value.BulkDataURI,
+                StudyInstanceUID: naturalized.StudyInstanceUID,
```

Comment:
The study UID is required if the BulkDataURI is relative, which is true for the back end I'm testing against.